flow2ml/Data_Augumentation.py

```diff
@@ -13,6 +18,24 @@ def __init__(self,techniques):
     '''
     self.techniques = techniques
 
-  ####### Note #######
-  ''' To be completed '''
-  ####### Note #######
\ No newline at end of file
+  def applyFlip(self,classPath):
+    ''' 
+      Applies flipping augmentation to all the images in the given folder. 
+      Args : 
+        classPath : (string) directory containing images for a particular class.
+    '''
+    try:
+      os.mkdir(classPath+"/FlippedImages")    
+    except:
+      pass
```

Comment:
Complete this exception block with suitable throw() or a valid message instead of passing it